examples/nxt-screen.py

```diff
@@ -12,12 +14,16 @@
 this code basically reimplements what
 https://bricxcc.sourceforge.net/utilities.html -> "NeXTScreen"
 in a less obscure programming language ;-)
+
+arohrbacher, 2.1.2024: Added code to provide a GUI representing the NXT physical buttons so that the NXT 
+brick can be remotely controlled
+
```

Comment:
This does not need to be in source file, it is provided in history.

Reply:
removed lines 18 and 19